app/src/main/java/com/github/alfabravo2013/readyforexams/domain/login/LoginRemoteDataSource.kt

```diff
@@ -0,0 +1,33 @@
+package com.github.alfabravo2013.readyforexams.domain.login
+
+import android.util.Log
+import com.github.alfabravo2013.readyforexams.util.Result
+import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.auth.FirebaseUser
+import com.google.firebase.auth.ktx.auth
+import com.google.firebase.ktx.Firebase
+
+class LoginRemoteDataSource {
+    private val auth: FirebaseAuth by lazy { Firebase.auth }
+
+    private var currentUser: FirebaseUser? = auth.currentUser
+
+    fun login(email: String, password: String): Result {
```

Comment:
I think you need to format this class as the return : for the function is not formatted properly.